app.py

```diff
@@ -57,17 +58,20 @@ def verify():
 #app route for verify check page
 @app.route('/verify/check/<certifycode>', methods=['GET', 'POST'])
 def verifycheck(certifycode):
+  image_exists_in_database = generate.retrieve_image_from_database(certifycode)
+  # The following line has become obsolete, verification is handeled through the database with the 'retrieve_image_from_database' funktion
   arr = glob.glob("static/verified/images/" + certifycode + ".*")
+
   #check if arr is not empty (means a file with same name exists)
-  if arr:
-    
+  if arr: # Possibly replace the following with 'image_exists_in_database'
+
     #take first image
     #split image path and get the end file name
     image = arr[0].split('/')[-1]
```

Comment:
what's the use of local file system now?

Reply:
Essentially just to be able to display the images through Flask.
I guess the `arr` variable is obsolete now.

Reply:
shouldn't we search Mlab database for verified images instead of local fs

Reply:
Good point.
Since the `retrieve_image_from_database` searches for the verified images in the database and already returns a True or False based on if it's in there it could just be replace it with that.
I mean yeah, I'd  recommend just replacing the 
`  if arr: 
image = arr[0].split('/')[-1]`
with 
`if image_exists_in_database:`

Hope this is what you mean ; D

Reply:
Alright. Let me test it then we can merge it.
Are you in the telegram channel?

Reply:
Nope, what is it?

Reply:
*Also sounds good 👍

Reply:
https://t.me/joinchat/F5h-ahGgim_LaqNrLUfliA

Reply:
And I'm in 👍